### ISSUE TITLE
Can't minify logical OR assignments

### ISSUE THREAD

**Issue by chzager** [olback/es6-css-minify#147]
**Describe the bug**
When trying to minify a JavaScript file where _logical OR assignments_ ([see MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR_assignment)) are used, Minify runs into an error ("Unexpected token: operator (=)")

**To Reproduce**
1. Make a new JS file
2. Sample code:
```JS
let a = "";
let b = "";
b ||= a;
````
3. Minify the file.
4. See error output console.

**Expected behavior**
The file should minify to
```JS
let a="",b="";b||=a;
```

Basically it is the same as #144.

**Desktop:**
VS-Code Version: 1.70.2
Minify Version: v3.3.3
OS: Windows_NT x64 10.0.18363
